### DISCUSSION TITLE
v1.21.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#3227] [Releases & Announcements]
## Changelog

- Instagram Channel
- Out of office autoresponder
- Reports for inbox, labels, agents, and teams
- Support cc and bcc for email channels
- Slack Integration supports attachments
- Ability to merge contacts
- Ability to delete contacts
- Conversation custom attribute APIs
- Ability to close and open chat widget via SDK
- Ability to import contacts
- Support for attachment messages in telegram and line channels
- Numerous bug fixes and enhancements

Thanks to @brentco @murtaza-bagwala @sarhugo @akhilgkrishnan @srameshr  @manojmj92 @seagalputra @SanjuSudheerM @WevrLabs for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v1.21.0'>v1.21.0</a>.</em>

**Comment by helpobras** [comment]
i use self hosted on ubuntu VPS. after updating gave error 400. the server no longer responds. by IP:3000 works.

**Reply by sojan-official** [reply]
https://github.com/chatwoot/chatwoot/issues/3228

**Reply by helpobras** [reply]
> [# 3228](https://github.com/chatwoot/chatwoot/issues/3228)

ok..muito obrigado.
